libs/app-localize/utils/src/lib/locales/en-US.json

```diff
@@ -1,4 +1,9 @@
 {
+  "general.components.devicesDrawer.title": "Select a device",
+  "general.components.deviceCard.recoveryModeLabel": "Recovery Mode",
+  "general.components.deviceCard.serialNumberLabel": "Serial number",
+  "general.components.deviceSelector.title": "Select a device to continue",
+  "general.components.deviceSelector.subtitle": "You can switch between connected devices later",
```

Comment:
⚪ 
The previous version: “You can change active devices later.” Has the revised copy been confirmed and updated on the figma?

Reply:
Good point, I must used wrong screen from Figma